maintenance/scripts/setup.sh

```diff
@@ -29,5 +25,7 @@ cp ./apps/web/.env.example ./apps/web/.env --preserve=all
 
 # build projects
 cd /var/www/html
-npm i
+npm ci
+./node_modules/.bin/cypress install
```

Comment:
Was this part of the fix? I think this might be a hack that means something is still happening that we don't understanding, as I'm pretty certain that the cypress action is supposed to handle this (and manage caching)

Reply:
Yeah, it was part of the fix. When we got to the cypress binary install it had problems because of incorrect permissions so moving it to the setup fixed that. I did not realize we were explicitly not installing it 🤦‍♀️ .

I removed this line and the line were we force it to install in the setup script and things seem to be fine. What permissions issues was the note referring to and will this cause more harm than good?

[attempt to pre-install cypress binary](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5173/commits/c087da7afe3d700d76f0ecd7722b1dd563d7641e)